index.js

```diff
@@ -41,16 +41,16 @@ function walkSync(baseDir, _options) {
     };
   }
 
-  return _walkSync(baseDir, options).map(mapFunct);
+  return _walkSync(baseDir, options, null, Object.create(null)).map(mapFunct);
```

Comment:
Just curious, have you thought about a Set here instead (we don’t really need a “value” per-se)? Any insight on why you preferred a `Object.create(null)`?

Reply:
this library supports node 0.10 still...

Reply:
Ah, node 0.10 has a set..

Reply:
😭 so when we drop it, should we experiment with using a Set?

Reply:
ya